tests/e2e_test.rs

```diff
@@ -231,37 +246,19 @@ async fn e2e() -> Result<()> {
     send_blobs_and_proofs(&client_node1).await?;
 
     // Wait for some slots to be finished
-    sleep(time::Duration::from_secs(15)).await;
+    sleep(time::Duration::from_secs(10)).await;
```

Comment:
I have a better idea to improve this for later: having a function that polls the latest block, and loop on this until a certain block is found. This way we guarantee no flakyness on the test whatever the speed of the machine running it

Reply:
for fixing later je pense